### DISCUSSION TITLE
Add settings for connections for Moonshine models and mirgations

### DISCUSSION THREAD

**Discussion by ramir1** [moonshine-software/moonshine#1114] [Ideas]
I would like to store Admin users in another database. For example, SQLite will be able to handle problems with the main database.

**Comment by lee-to** [comment]
No one is stopping you from specifying different connections in the models, this is part of the framework and not moonshine, if problems arise, then create an issue with a detailed description of how to reproduce, what errors, and so on

**Reply by ramir1** [reply]
Имел ввиду для моделей из Муншайн: moonshine_users, moonshine_user_roles и т.д.
Что-то вроде:
`Schema::connection(config('moonshine.db'))
->create('moonshine_user_roles', static function (Blueprint $table):`
и в моделях..
